packages/appstore/src/constants/routes-config.ts

```diff
@@ -2,6 +2,7 @@ import { localize } from '@deriv/translations';
 import TradingHub from 'Modules/trading-hub';
 import ConfigStore from 'Stores/config-store';
 import { TRoute } from 'Types';
+import Onboarding from 'Components/onboarding';
```

Comment:
add module for onboarding in the same directory and move the onboarding component there
```suggestion
import Onboarding from 'Modules/onboarding';
```

Reply:
Will do 👀